Ninja generated error when setting PCAP option in west

**Describe the bug**
When building Zephyr with following west options
```
west build -b qemu_x86 -d build/server samples/net/sockets/echo_server -- -DOVERLAY_CONFIG=overlay-qemu_802154.conf -DPCAP=capture.pcap
west build -t server -d build/server
```
The output is
```
-- west build: running target server
[0/1] To exit from QEMU enter: 'CTRL+a, x'[QEMU] CPU: qemu32,+nx,+pae
/bin/sh: -c: line 0: syntax error near unexpected token `&&'
/bin/sh: -c: line 0: `cd zephyr/build/server && cmake/cmake-3.13.1-Linux-x86_64/bin/cmake -E remove -f /tmp/ip-stack-server.in /tmp/ip-stack-server.out /tmp/ip-stack-client.in /tmp/ip-stack-client.out && mkfifo /tmp/ip-stack-server.in && mkfifo /tmp/ip-stack-server.out && mkfifo /tmp/ip-stack-client.in && mkfifo /tmp/ip-stack-client.out && zephyr/../net-tools/monitor_15_4 capture.pcap /tmp/ip-stack-server /tmp/ip-stack-client > /dev/null & && /opt/zephyr-sdk/sysroots/x86_64-pokysdk-linux/usr/bin/qemu-system-i386 -m 9 -cpu qemu32,+nx,+pae -device isa-debug-exit,iobase=0xf4,iosize=0x04 -no-reboot -nographic -net none -pidfile qemu.pid -chardev stdio,id=con,mux=on -serial chardev:con -mon chardev=con,mode=readline -serial pipe:/tmp/ip-stack-server -pidfile qemu-server.pid -kernel zephyr/build/server/zephyr/zephyr.elf'
FAILED: zephyr/CMakeFiles/server 
cd zephyr/build/server && cmake/cmake-3.13.1-Linux-x86_64/bin/cmake -E remove -f /tmp/ip-stack-server.in /tmp/ip-stack-server.out /tmp/ip-stack-client.in /tmp/ip-stack-client.out && mkfifo /tmp/ip-stack-server.in && mkfifo /tmp/ip-stack-server.out && mkfifo /tmp/ip-stack-client.in && mkfifo /tmp/ip-stack-client.out && zephyr/../net-tools/monitor_15_4 capture.pcap /tmp/ip-stack-server /tmp/ip-stack-client > /dev/null & && /opt/zephyr-sdk/sysroots/x86_64-pokysdk-linux/usr/bin/qemu-system-i386 -m 9 -cpu qemu32,+nx,+pae -device isa-debug-exit,iobase=0xf4,iosize=0x04 -no-reboot -nographic -net none -pidfile qemu.pid -chardev stdio,id=con,mux=on -serial chardev:con -mon chardev=con,mode=readline -serial pipe:/tmp/ip-stack-server -pidfile qemu-server.pid -kernel zephyr/build/server/zephyr/zephyr.elf
ninja: build stopped: subcommand failed.
FATAL ERROR: command exited with status 1: cmake/cmake-3.13.1-Linux-x86_64/bin/cmake --build zephyr/build/server --target server
```

**Expected behavior**
There is no error like this. If I use `make` then everything is compiled correctly
```
west build -b qemu_x86 -d build/server samples/net/sockets/echo_server -- -G'Unix Makefiles' -DOVERLAY_CONFIG=overlay-qemu_802154.conf -DPCAP=capture.pcap
west build -t server -d build/server
-- west build: running target server
[  2%] Built target parse_syscalls_target
[  3%] Built target syscall_list_h_target
[  4%] Built target kobj_types_h_target
[  5%] Built target driver_validation_h_target
[  5%] Built target offsets
[  6%] Built target offsets_h
[  6%] Built target zephyr_generated_headers
[ 19%] Built target kernel
[ 19%] Built target gen_echo-apps-cert_der_inc_bfdf056d50a2f75b6de35e03cf03ab35
[ 19%] Built target gen_server_privkey_der_inc_8d0e98f4d185497b7169671e6654093b
[ 20%] Built target gen_echo-apps-key_der_inc_71cdf4a8da5c7ae742e6cea12b434869
[ 21%] Built target gen_server_der_inc_011ce3683e31d2b8920db88fd98b32f6
[ 22%] Built target gen_ca_der_inc_69fc43a25d0dca3371fea6a42d00c7c8
[ 23%] Built target app
[ 51%] Built target zephyr
[ 52%] Built target linker_script_target
[ 61%] Built target arch__x86__core
[ 69%] Built target lib__libc__minimal
[ 71%] Built target lib__posix
[ 72%] Built target subsys__net
[ 75%] Built target subsys__net__l2__ieee802154
[ 85%] Built target subsys__net__ip
[ 87%] Built target subsys__net__lib__config
[ 88%] Built target subsys__net__lib__conn_mgr
[ 89%] Built target subsys__random
[ 90%] Built target drivers__serial
[ 92%] Built target zephyr_prebuilt
[ 92%] Built target linker_pass_final_script_target
[ 92%] Built target gen_idt_output
[ 93%] Built target staticIdt_o
[ 95%] Built target irq_vectors_alloc_o
[ 95%] Built target pagetables_bin_target
[ 96%] Built target pagetables_o
[ 97%] Built target irq_int_vector_map_o
[ 98%] Built target zephyr_final
Scanning dependencies of target server
[100%] 
To exit from QEMU enter: 'CTRL+a, x'
[QEMU] CPU: qemu32,+nx,+pae
SeaBIOS (version rel-1.12.1-0-ga5cab58-dirty-20200516_152813-e7ee184eb100-zephyr
)
Booting from ROM..*** Booting Zephyr OS build zephyr-v2.4.0-1229-g0d216924d5cd  ***
```

**Impact**
Annoyance mostly.

**Environment:**
 - OS: Linux only (does not work in Windows)

--- a/cmake/emu/qemu.cmake
+++ b/cmake/emu/qemu.cmake
@@ -226,11 +226,17 @@ elseif(QEMU_NET_STACK)
     set_ifndef(NET_TOOLS ${ZEPHYR_BASE}/../net-tools) # Default if not set
 
     list(APPEND PRE_QEMU_COMMANDS_FOR_server
-      COMMAND ${NET_TOOLS}/monitor_15_4
-  ${PCAP}
-  /tmp/ip-stack-server
+      COMMAND
+      #This command is run in the background using '&'. This prevents
+      #chaining other commands with '&&'. The command is enclosed in '{}'
+      #to fix this.
+      {
+      ${NET_TOOLS}/monitor_15_4
+      ${PCAP}
+      /tmp/ip-stack-server
       /tmp/ip-stack-client
       > /dev/null &
+      }
       # TODO: Support cleanup of the monitor_15_4 process
       )
   endif()

